storage/store.go

```diff
@@ -88,18 +81,7 @@ func (s *etcdStore) Start(ctx context.Context) error {
 	return nil
 }
 
-func (s *etcdStore) Put(ctx context.Context, job *JobRecord, options JobRecordOptions) error {
-	opts := []etcdclient.OpOption{}
-	if options.TTL > 0 {
-		// Create a lease
```

Comment:
This is a performance optmization that allows writes on etcd not to require a lease per key with ttl. Deletion of expired jobs is handled by our own cron logic now.